tools/stats/s3_stat_parser.py

```diff
@@ -210,9 +224,10 @@ def get_previous_reports_for_branch(branch: str, ci_job_prefix: str = "") -> Lis
         logger.info(f'Grabbing reports from commit: {commit}')
         summaries = get_test_stats_summaries_for_job(sha=commit, job_prefix=ci_job_prefix)
         for job_name, summary in summaries.items():
-            reports.append(summary[0])
-            if len(summary) > 1:
-                logger.warning(f'WARNING: Multiple summary objects found for {commit}/{job_name}')
+            if job_name[:len(job_name) - len('-test')] == _get_stripped_CI_job():
```

Comment:
could you switch this to a more explicit "if job_name doesn't have slow" and add a comment for why we should exclude slow tests? -->Also wouldn't this mean the slow tests would have no stats for sharding?